manifests/common/crd/hardwareprofiles.opendatahub.io.crd.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: hardwareprofiles.dashboard.opendatahub.io
+spec:
+  group: dashboard.opendatahub.io
+  scope: Namespaced
+  names:
+    plural: hardwareprofiles
+    singular: hardwareprofile
+    kind: HardwareProfile
+  versions:
+    - name: v1
```

Comment:
can you make this v1alpha1 for now. we may have to do changes to the crd so better have it be in alpha